pulseapi/urls.py

```diff
@@ -25,27 +25,31 @@
     RSSFeedFeaturedFromPulse,
     AtomFeedFeaturedFromPulse
 )
+from pulseapi.utility.urlutils import (
+    versioned_url,
+    versioned_api_url,
+)
 
 urlpatterns = [
     # admin patterns
     url(r'^admin/', admin.site.urls),
 
     # 'homepage'
-    url(r'^', include('pulseapi.users.urls')),
+    url(versioned_url(r'^'), include('pulseapi.users.urls')),
 
     # API routes
-    url(r'^api/pulse/', include('pulseapi.users.urls')),
-    url(r'^api/pulse/entries/', include('pulseapi.entries.urls')),
-    url(r'^api/pulse/profiles/', include('pulseapi.profiles.urls')),
-    url(r'^api/pulse/tags/', include('pulseapi.tags.urls')),
-    url(r'^api/pulse/issues/', include('pulseapi.issues.urls')),
-    url(r'^api/pulse/helptypes/', include('pulseapi.helptypes.urls')),
-    url(r'^api/pulse/creators/', include('pulseapi.creators.urls')),
+    url(versioned_api_url(), include('pulseapi.users.urls')),
+    url(versioned_api_url(r'entries/'), include('pulseapi.entries.urls')),
```

Comment:
we're using forward slashes, not backslashes, so do we need the raw string prefix here?

Reply:
nah it's not needed I can remove